app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -0,0 +1,55 @@
+module MiddlewareServerHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    @prop_hash = eval(@record.properties)
```

Comment:
Right now that record.properties is a json structure that we store in Hawkular as json. While possible keys are limited, I am not convinced that putting each property into its own column is the best way to implement. What else can we do. Especially in the light of JSON native support in postgres9.5 (?)